lib/CoreDisTools/coredistools.exports

```diff
@@ -0,0 +1,3 @@
+InitDisasm
+FinishDisasm
+DisasmInstruction
```

Comment:
It appears PrintInstruction is an export in the header file but you didn't specify it here.